test/browser/render.test.js

```diff
@@ -815,6 +815,42 @@ describe('render()', () => {
 		expect(scratch.innerHTML).to.contain(`<span>${todoText}</span>`);
 	});
 
+	it.skip('equal vnodes should short-circuit diffing', () => {
```

Comment:
Is this skipped by accident or does the test fail?

Reply:
This is the test I prepared for the setState one, this fails for now as I can't find out how to properly tackle this yet.